packages/cli/src/lib/commands/collect/utils/persist/open-report.unit.test.ts

```diff
@@ -0,0 +1,108 @@
+import * as openReport from 'open';
+import { handleOpenFlowReports, openFlowReports } from './open-report';
+import { logVerbose } from '../../../../core/loggin';
+import { CollectCommandOptions } from '../../options';
+
+describe('handleOpenFlowReport', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  })
+
+  it('should return the openFlowReport function if openReport, interactive and not dryRun', async () => {
+    const openReportsProcess = handleOpenFlowReports({
+      openReport: true,
+      interactive: true,
+      dryRun: false,
+    } as CollectCommandOptions);
+    expect(typeof openReportsProcess).toEqual("function");
```

Comment:
Jest has a special [matcher](https://archive.jestjs.io/docs/en/22.x/expect#expectanyconstructor) for this.
```suggestion
    expect(openReportsProcess).toEqual(expect.any(Function));
```